packages/styles/src/components/tag.scss

```diff
@@ -0,0 +1,53 @@
+@forward './../variables/options';
+@forward './utilities';
+
+@use './../variables/components/tag' as tag;
+
+.tag:has(:not(pre)), // is only a temporary fix till I know how to do it wihtout causing future issues. wihtout this the Code sample would break currently.
+.tag-sm {
+  display: inline-flex;
+  justify-content: flex-start;
+  justify-items: center;
```

Comment:
The `justify-items` property is for grid containers, it is ignored in flexbox layouts: https://developer.mozilla.org/en-US/docs/Web/CSS/justify-items?utm_source=devtools&utm_medium=inspector-inactive-css
```suggestion
```